### PR TITLE
[ACTION] PandaDoc - Create Document: add optional prop for folder

### PR DESCRIPTION
**Pull request by zalmanlew**

### CODE REVIEW

**Comment by pipedream-component-development** [comment]
Thank you so much for submitting this! We've added it to our backlog to review, and our team has been notified.

**Comment by pipedream-component-development** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:
- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/pandadoc/actions/create-document-from-file/create-document-from-file.mjs [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default {
   name: "Create Document From File",
   description: "Create a document from a file or public file URL. [See the docs here](https://developers.pandadoc.com/reference/create-document-from-pdf)",
   type: "action",
-  version: "0.0.1",
+  version: "0.0.2",
```
Comment (dylburger) [inline components/pandadoc/actions/create-document-from-file/create-document-from-file.mjs:10]:
```suggestion
  version: "0.0.2",
```

Comment (dylburger) [reply components/pandadoc/actions/create-document-from-file/create-document-from-file.mjs:10]: Removed unnecessary double quotes

File: components/pandadoc/actions/create-document-from-file/create-document-from-file.mjs [hunk 2]
```diff
@@ -35,6 +35,12 @@ export default {
       description: "A public file URL to use instead of a local file.",
       optional: true,
     },
+    documentFolderId: {
+      propDefinition: [
+        app,
+        "documentFolderId",
```
Comment (dylburger) [inline components/pandadoc/actions/create-document-from-file/create-document-from-file.mjs:41]:
```suggestion
        "documentFolderId",
```

Comment (dylburger) [reply components/pandadoc/actions/create-document-from-file/create-document-from-file.mjs:41]: Removed unnecessary double quotes

File: components/pandadoc/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/pandadoc",
-  "version": "0.1.0",
+  "version": "0.1.1",
```
Comment (dylburger) [inline components/pandadoc/package.json:3]:
```suggestion
  "version": "0.1.1",
```

Comment (dylburger) [reply components/pandadoc/package.json:3]: Removed unnecessary double quotes

**Review by vellames — APPROVED** [review]

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[PandaDoc_6023_0004.pdf](https://github.com/PipedreamHQ/pipedream/files/11235357/PandaDoc_6023_0004.pdf)

**Pull request merged**